### DISCUSSION TITLE
Tutorial for patching libre 3 by ourselfs needed

### DISCUSSION THREAD

**Discussion by twinko** [maheini/FreeStyle-Libre-3-patch#21] [Ideas]
Hey,

at least for me its pretty concerning that juggluco isnt available in the playstore anymore. At least for me i have the felling the patched libre 3 app will be the next thing removed.
I think we should take this as a first warning and write up a tutorial on how to patch the libre 3 app by ourselfs. This looks like its legally fine, if we concider that this solution is still available for the libre 2 app.

Wpuld be great if this would be a backup solution. If i can help, count me in!

**Comment by maheini** [comment]
Hey twinko

It is a little bit concering, however, I don't think it's a problem. If abbott really wants to shut down all solutions, I guess they would have contacted me already. And even if they really are forcing me to remove this repo, there are a lot of forks and the apks are already widely distributed.

It's pretty complex to patch libre 3 by yourself and jkaltes would have to share his solution. I think as long as he isn't willing to share informations, it doesn't matter. Our current backup is the already widely shared patch of libre 3.
